dts/arm/nordic/nrf51822_qfaa.dtsi

```diff
@@ -14,3 +14,9 @@
 &sram0 {
 	reg = <0x20000000 DT_SIZE_K(16)>;
 };
+
+/ {
+	soc {
+		compatible = "nrf,nRF51822-QFAA", "nrf,nRF51822", "nrf,nRF51", "simple-bus";
```

Comment:
I suppose, "simple-bus" is not needed, as it is added in the ARM core .dtsi. Or...?

Reply:
simple-bus is needed here because dts doesn't merge, it replaces.